prov/util/src/util_mem_monitor.c

```diff
@@ -66,19 +75,30 @@ void ofi_monitor_init(void)
 			" memory footprint, but can negatively impact"
 			" performance in some situations.  (default: false)");
 
+	fi_param_define(NULL, "core_monitor", FI_PARAM_BOOL,
+			"TODO: Need the description here. Choose core monitor");
+
```

Comment:
Drop this for now.  We can manually set the default for testing and add a parameter to set the default after merging.